### PR TITLE
Fix header file missing (which would cause compile error on Fedora OS)

### PR DESCRIPTION
**Pull request by kwbm**
Fix header file missing (which would cause compile error on Fedora OS)

### CODE REVIEW

**Comment by hyv1001** [comment]
The `matrix3.h` and `matrix4.h` don't use the std::string, could you create an issue about the compile error?

**Comment by kwbm** [comment]
Screenshot from 2022-04-06 23-53-13

**Comment by kwbm** [comment]
missing "memcpy" declatations

**Comment by wlbksy** [comment]
IMO, this should be <cstring> as this is a cpp project instead of c project.

File: engine/source/runtime/core/math/matrix3.h [hunk 1]
```diff
@@ -4,6 +4,8 @@
 #include "runtime/core/math/quaternion.h"
 #include "runtime/core/math/vector3.h"
 
+#include <string.h>
```
Comment (hyv1001) [inline engine/source/runtime/core/math/matrix3.h:7]: please use #include <cstring>

File: engine/source/runtime/core/math/matrix4.h [hunk 2]
```diff
@@ -6,6 +6,8 @@
 #include "runtime/core/math/vector3.h"
 #include "runtime/core/math/vector4.h"
 
+#include <string.h>
```
Comment (hyv1001) [inline engine/source/runtime/core/math/matrix4.h:9]: <cstring>

**Review by ShenMian — APPROVED** [review]

**Pull request merged**